From cecf3efeb0f6b461447d833ffb4620f64e48c7e0 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Tue, 14 Feb 2017 12:17:12 +0000
Subject: [PATCH 2/3] mx6sx: udoo_neo: use different load address for ramdisk

The fdt_addr and ramdisk_addr_r are currently both defined to
0x83000000 and that's not going to work well for anyone. Move
the ramdisk_addr_r to 0x84000000.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 include/configs/udoo_neo.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/udoo_neo.h b/include/configs/udoo_neo.h
index 5b46272e..1d737cc 100644
--- a/include/configs/udoo_neo.h
+++ b/include/configs/udoo_neo.h
@@ -52,7 +52,7 @@
 			"echo WARNING: Could not determine dtb to use; fi\0" \
 	"kernel_addr_r=" __stringify(CONFIG_LOADADDR) "\0" \
 	"pxefile_addr_r=" __stringify(CONFIG_LOADADDR) "\0" \
-	"ramdisk_addr_r=0x83000000\0" \
+	"ramdisk_addr_r=0x84000000\0" \
 	"scriptaddr=" __stringify(CONFIG_LOADADDR) "\0" \
 	BOOTENV
 
-- 
2.9.3

